bedrock/firefox/views.py

```diff
@@ -541,6 +542,18 @@ def get_template_names(self):
         return [template]
 
 
+def firefox_os_index(request):
```

Comment:
So the reason not to change this view name was that there's already a function with this name up on line 228. Either delete that one or (preferably) just move this one up and replace that one.